### PR TITLE
Data science publisher landing page

### PR DESCRIPTION
**Pull request by MxRaCohen**
Initializes a landing page for data science publishers

### CODE REVIEW

File: ethicalads-theme/templates/ea/landing-pages/data-science.html [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+  <!-- Content -->
+  <div class="container">
+    <div class="row align-items-center text-center text-md-left">
+      <div class="col-12 col-md-6">
+
+        <!-- Image -->
+        <img src="/theme/img/data-code.jpg" alt="Data Code" class="img-fluid mw-100 float-right mb-6 mb-md-0" data-aos="fade-right">
+        <p class="small text-muted">
+          <em>
+          Photo by <a href="https://unsplash.com/@chrisliverani?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Chris Liverani</a> on <a href="https://unsplash.com/s/photos/big-data?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/landing-pages/data-science.html:24]: Love this image. 💯

File: ethicalads-theme/templates/ea/landing-pages/data-science.html [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+        <!-- Icon -->
+        <div class="icon text-primary mb-3">
+          <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M0 0h24v24H0z"></path><path fill="#335EEA" opacity=".3" d="M12 20.66l-2.536-.025-2.146-1.35-2.12-1.391-1.075-2.297-1.031-2.316.336-2.513.385-2.507L5.455 6.33l1.68-1.9L9.56 3.69 12 3l2.44.69 2.426.739 1.679 1.9 1.642 1.932.385 2.507.336 2.513-1.03 2.316-1.076 2.297-2.12 1.391-2.146 1.35z"></path><circle fill="#335EEA" opacity=".3" cx="8.5" cy="13.5" r="1.5"></circle><circle fill="#335EEA" opacity=".3" cx="13.5" cy="7.5" r="1.5"></circle><circle fill="#335EEA" opacity=".3" cx="14.5" cy="15.5" r="1.5"></circle></g></svg>
+        </div>
+
+        <!-- Heading -->
+        <h3>No Cookies</h3>
+
+        <!-- Text -->
+        <p class="text-gray-800 mb-6 mb-md-0">We don't use cookies to track users, but we match ads based on the content of your site. This gives your users <strong>ultimate privacy and relevant ads</strong>, the best of both worlds.</p>
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/landing-pages/data-science.html:78]: I wonder about changing up the copy on this point and the previous one? They are the same as the python ones.

File: ethicalads-theme/templates/ea/landing-pages/data-science.html [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+
+
+<!-- Sample Ad -->
+<!-- ================================================== -->
+<section class="py-10">
+  <div class="container">
+    <div class="row justify-content-center">
+
+      <div class="col-md-6 mb-3">
+        <h2 class="font-weight-bold mb-8">Data driven</h2>
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/landing-pages/data-science.html:107]: Should we aim for another set of keywords here? It's the same as the `Driven by data` above.

File: ethicalads-theme/templates/ea/landing-pages/data-science.html [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+    {% endwith %}
+  </div> <!-- / .container -->
+</section>
+
+<!-- Social proof -->
+<!-- ================================================== -->
+<section class="container py-10">
+  <div class="row">
+    <div class="col mb-3">
+      <h2 class="text-center font-weight-bold mb-8">Over 100 happy publishers</h2>
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/landing-pages/data-science.html:130]: Should we put the keyword in the h2 here? This one doesn't buy us much, and I copied it as well :D

File: ethicalads-theme/templates/ea/landing-pages/data-science.html (hunk 3 above)
Comment (MxRaCohen) [reply ethicalads-theme/templates/ea/landing-pages/data-science.html]: I expanded it to include discrete although I'm not sure how popular that keyword is in relevant searches.

File: ethicalads-theme/templates/ea/landing-pages/data-science.html [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+    {% endwith %}
+  </div> <!-- / .container -->
+</section>
+
+<!-- Social proof -->
+<!-- ================================================== -->
+<section class="container py-10">
+  <div class="row">
+    <div class="col mb-3">
+      <h2 class="text-center font-weight-bold mb-8">Content with content-targeting</h2>
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/landing-pages/data-science.html:130]: This seems clever, but I don't think people will understand it. It took me a couple reads to understand it wasn't a typo. 

To keep a bit of alliteration, perhaps:

```suggestion
      <h2 class="text-center font-weight-bold mb-8">Proud of our privacy</h2>
```

**Pull request merged**